exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -143,6 +151,7 @@ func newPRWExporter(cfg *Config, set exporter.Settings) (*prwExporter, error) {
 		batchTimeSeriesState: newBatchTimeSericesState(),
 	}
 
+	prwe.settings.Logger.Info("Running with prometheus remote write proto message", zap.Any("ProtoMsg", cfg.RemoteWriteProtoMsg))
```

Comment:
```suggestion
	prwe.settings.Logger.Info("starting prometheus remote write exporter", zap.Any("ProtoMsg", cfg.RemoteWriteProtoMsg))
```

We don't need the elaborate string, because there will be "ProtoMsg=<proto mess>" in the log?